mobile/src/translations/ru.json

```diff
@@ -75,6 +75,17 @@
     "screens": {
       "loading": {
         "loadingText": "Загрузка..."
+      },
```

Comment:
nice thing, will translate it sometime later

Reply:
You rock 👯‍♂️